src/main/java/com/wire/bots/hold/model/EventModel.java

```diff
@@ -0,0 +1,9 @@
+package com.wire.bots.hold.model;
+
+import com.wire.bots.hold.model.database.Event;
+
+import java.util.List;
+
+public class EventModel {
+    public List<Event> events;
+}
```

Comment:
There was a duplicate class named `Model` being created and duplicated across other files, so I just added to a proper file to be used.